src/activities/anuj_activity/Anuj_activity.qml

```diff
@@ -0,0 +1,202 @@
+ *   the Free Software Foundation; either version 3 of the License, or
+ *   (at your option) any later version.
+ *
+ *   This program is distributed in the hope that it will be useful,
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *   GNU General Public License for more details.
+ *
+ *   You should have received a copy of the GNU General Public License
+ *   along with this program; if not, see <https://www.gnu.org/licenses/>.
+ */
+import QtQuick 2.6
+import QtQuick.Controls 1.1
+
+import "../../core"
+import "anuj_activity.js" as Activity
+
+ActivityBase {
+    id: activity
+    x: 0
+
+    onStart: focus = true
+    onStop: {}
+
+    pageComponent: Rectangle {
+        id: background
+        anchors.fill: parent
+        color: "white"
+        signal start
+        signal stop
+
+        Component.onCompleted: {
+            activity.start.connect(start)
+            activity.stop.connect(stop)
+        }
+
+        // Add here the QML items you need to access in javascript
+        QtObject {
+            id: items
+            property Item main: activity.main
+            property alias background: background
+            property alias bar: bar
+            property alias bonus: bonus
+            property alias header: header
+            property alias rocket: rocket
+            property alias mountain : mountain
+            property alias launchbutton: launchbutton
+        }
+
+        onStart: { Activity.start(items) }
+        onStop: { Activity.stop() }
+
+        DialogHelp {
+            id: dialogHelp
+            onClose: home()
+        }
+
+        Rectangle {
+            id: header
+            width: parent.width
+            height: headerText.height
+            anchors.top: parent.top
+            anchors.topMargin: 10
+            border.width: 2
+            border.color: "black"
+            color: "blue"
+            z: 5
+
+            property alias headerText: headerText
+
+            GCText {
+                id: headerText
+                anchors.centerIn: parent
+                width: parent.width * 3 / 4
+                fontSize: mediumSize
+                wrapMode: Text.WordWrap
+                horizontalAlignment: Text.AlignHCenter
+                text: qsTr("Click on the button to shoot the ROCKET! and Click on Sun to change Time of Day !")
+                color: "white"
+            }
+        }
+
+
+                Image {
+                    id : mountain
+                source: "../anuj_activity/resource/mountain.png"
+                anchors {
+                    left: parent.left
+                    right: parent.right
+                    bottom: parent.bottom
+                }
+                height : parent.height * 0.8
+              }
+
+
+
+            Image {
+                id: rocket
+                x : 0
+                y : 300
```

Comment:
positions should not be hardcoded, the resolution of the device may change (depending on a phone, table, screen...)